test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetListPackageTests.cs

```diff
@@ -210,7 +210,7 @@ public void DotnetListPackage_DeprecatedAndOutdated_Fail()
                     true);
 
                 Assert.False(listResult.Success);
-                Assert.Contains(Strings.ListPkg_InvalidOptionsOutdatedAndDeprecated, listResult.Errors);
+                Assert.Contains("Option '--outdated' and '--deprecated' cannot be combined.", listResult.Errors);
```

Comment:
Why is this string slightly different, and can they both be the same and used in lieu of the hard coded string in the test? 
`ListPkg_InvalidOptionsOutdatedAndDeprecated = "Invalid command. Combining the '--outdated' and '--deprecated' options is not supported"`

Reply:
That's what got added in the SDK. 
I will create a separate issue to align these.

Reply:
https://github.com/NuGet/Home/issues/8454

Reply:
PR out to fix the messaging: https://github.com/NuGet/NuGet.Client/pull/3013

Reply:
The PR has been merged. Can we go back to the non-literal now?